tests/test_MB_dist.py

```diff
@@ -0,0 +1,5 @@
+from MB_dist import function_name
+
+def test_function_name():
```

Comment:
```suggestion
def test_function_name():
```
Could add docstring to explain purpose of test